pallets/registrar/src/lib.rs

```diff
@@ -598,11 +633,8 @@ pub mod pallet {
         pub fn is_para_manager(para_id: &ParaId, account: &T::AccountId) -> bool {
             // This check will only pass if both are true:
             // * The para_id has a deposit in pallet_registrar
-            // * The deposit creator is the signed_account
-            RegistrarDeposit::<T>::get(para_id)
-                .map(|deposit_info| deposit_info.creator)
-                .as_ref()
-                == Some(account)
+            // * The signed_account is the para manager
```

Comment:
Since we are no longer checking the deposit, you can remove this comment, as the new logic is simple enough to follow without comments.

Reply:
I've updated the logic again to also check for the creator is the manager is not set (for whatever reason)